docs/job_log.md

```diff
@@ -0,0 +1,153 @@
+	"containerExitType" : "UNKNOWN"
+	}
+	[g2p_train]: [LastCompletedTask]: [ContainerLocations]:
+	ContainerLogHttpAddress: http://10.151.40.165:8042/node/containerlogs/container_e9878_1532412068340_0018_01_000002/core/
+	AppCacheNetworkPath: 10.151.40.165:/var/lib/hadoopdata/nm-local-dir/usercache/core/appcache/application_1532412068340_0018
+	ContainerLogNetworkPath: 10.151.40.165:/var/lib/yarn/userlogs/application_1532412068340_0018/container_e9878_1532412068340_0018_01_000002
+	________________________________________________________________________________________________________________________________________________________________________________________________________
+	[ApplicationCompletionReason]: [g2p_train]: FailedTaskCount 1 has reached MinFailedTaskCount 1.
+
+```
+
+Please pay attention to these lines to diagnostic job failure reason
+
+| line head | above example log info | 
+| --- | --- | 
+| [ExitDiagnostics] | ExitStatus undefined in Launcher, maybe UserApplication itself failed.| 
+| [ExitCode] | 134| 
+| Exception message | No such object: cntk-test-4621-17223-container_e9878_1532412068340_0018_01_000002. | 
+| Shell output | [DEBUG] EXIT signal received in yarn container, exiting ...[DEBUG] cntk-test-4621-17223-container_e9878_1532412068340_0018_01_000002 does not exist.| 
+|ContainerLogHttpAddress| ```http://10.151.40.165:8042/node/containerlogs/container_e9878_1532412068340_0018_01_000002/core/ ```|
+|AppCacheNetworkPath|10.151.40.165:/var/lib/hadoopdata/nm-local-dir/usercache/core/appcache/application_1532412068340_0018|
+|ContainerLogNetworkPath|10.151.40.165:/var/lib/yarn/userlogs/application_1532412068340_0018/container_e9878_1532412068340_0018_01_000002|
+|[ApplicationCompletionReason]| [g2p_train]: FailedTaskCount 1 has reached MinFailedTaskCount 1.|
+
+we could get information:
+1.  UserApplication itself failed.
+2.  cntk-test-4621-17223-container_e9878_1532412068340_0018_01_000002 does not exist is the reason.
+3.  Then we could visit ```http://10.151.40.165:8042/node/containerlogs/container_e9878_1532412068340_0018_01_000002/core/``` at step 1.2 to detect task failure reason.
+
+
+### 1.2 View job each task container log  <a name="tasklog"></a>
+
+- Check the failed task log who triggered the whole attempt failed, i.e. 
```

Comment:
>Check [](start = 2, length = 5)

Please add the information about log rotation, and refer to the corresponding configuration file (or doc if we have)

Reply:
Hi @ydye, currently, The default configuration of log rotation is at which file ? user could configure which items to control log rotation?